src/firefly/js/ui/tap/ObjectIDSearch.jsx

```diff
@@ -48,6 +48,10 @@ export function ObjectIDSearch({cols, initArgs={}, capabilities, tableName,colum
         }
     );
 
+    useEffect(() => {
+        if (uploadInfo?.columns) checkHeaderCtl.setPanelActive(true);
+    }, [uploadInfo]);
```

Comment:
here's the latest change @robyww to select the Object ID Search helper on a table upload. And the useFieldGroupWatch code is right above it.